### ISSUE TITLE
Media: navigation bar buttons add, edit are not descriptive and trash button is not accessible

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12874]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. In the WordPress app, go to My Site > Media.
1. Explore the buttons in the top bar for add, edit, and trash (after you go into edit mode).

**Result:** the Media navigation buttons could use better descriptions and the trash button is not accessible at all saying "Dim. Button" when it is inactive (while no media is selected) and just "Button" when it is active (while media is selected). While images are being deleted, and afterward, it's really easy to get lost with what's happening if you are navigating without sight. Here is a video with an example of deleting images while using VoiceOver: [1m12](https://cloudup.com/cBi15ozlA2Y).

> The edit button could have a hint saying how editing works (multi-select to delete), since it’s not clear what happens after activating it. 

<img src="https://user-images.githubusercontent.com/1119271/68255752-65b0e500-ffeb-11e9-8629-0432c439b39a.png" width="270" alt="media-edit-button"> <img src="https://user-images.githubusercontent.com/1119271/68255754-66497b80-ffeb-11e9-8b0f-f03bca1b5b4c.png" width="270" alt="media-trash-button-with-selected-images">
<sup>Tested with WPiOS 13.5.0.3 (App Store version) on iPhone 6S iOS 13.2.</sup>

<p style="clear:both;"><sub>/hat tip @etoledom for the report, internal reference: p77Llu-bQ6-p2</sub></p>
